lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -538,17 +538,14 @@ function collectAllStringsInDir(dir) {
     if (!process.env.CI) console.log('Collecting from', relativeToRootPath);
 
     const content = fs.readFileSync(absolutePath, 'utf8');
-    const exportVars = require(absolutePath);
     const regexMatch = content.match(UISTRINGS_REGEX);
-    const exportedUIStrings = exportVars.UIStrings;
-
     if (!regexMatch) {
-      // No UIStrings found in the file text or exports, so move to the next.
-      if (!exportedUIStrings) continue;
-
-      throw new Error('UIStrings exported but no definition found');
```

Comment:
I know I suggested this, but I think we don't want to get rid of this error because it catches non-standard `UIStrings` declarations that won't parse with tsc but not warn you about the error.

Should we just add some form of `standalone` to the `ignoredPathComponents`?

Reply:
> Should we just add some form of standalone to the ignoredPathComponents?

+1 if the goal here is to make this script work for standalone report only

I see a future in which we want to have this script work on files that work in the browser though in which case we need a longer term answer for both concerns. 

Alternatives: 

- we have two UIStrings regex, one that's simpler for *any* `UIStrings` use, and then our current definition one. We only skip import if there's no mention of `UIStrings`, period.
- we shim browser dependencies necessary to be able to import the keys at least